src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php

```diff
@@ -107,6 +108,14 @@ protected function getRequest(SentMessage $message): SendEmailRequest
             $request['FeedbackForwardingEmailAddress'] = $email->getReturnPath()->toString();
         }
 
+        if ($customHeaders = $this->getCustomHeaders($email->getHeaders())) {
+            if (\count($customHeaders) > 15) {
```

Comment:
I think we're doing such validation anywhere else as the provider will throw an error anyway.
Also, if the count changes to something else in the future, having the limit hardcoded here would be problematic (and we don't want to monitor such third-party changes).

Reply:
Done, removed this validation.